HermaFx.Foundation/RuntimeExtensions/GuidExtensions.cs

```diff
@@ -19,5 +21,15 @@ public static bool TryParse(string input, out Guid result)
 			result = default(Guid);
 			return false;
 		}
+
+		public static string ToZBase32String(this Guid guid)
+		{
+			return new ZBase32Encoder().Encode(guid.ToByteArray());
+		}
+
+		public static string ToBase64String(this Guid guid)
```

Comment:
This is missing an overload: ToBase64String(this Guid, bool trimmed) => if trimmed == true, then return should have padding (ie. '==') removed.